hibernate/cache-commons/src/test/java/org/infinispan/test/hibernate/cache/commons/functional/cluster/AbstractPartialUpdateTest.java

```diff
@@ -126,21 +125,15 @@ public AbstractInjectPartialFailure(Properties properties) {
          super(properties);
       }
 
-      private static int ENTITY_CONFIG_COUNTER = 0;
-
       @Override
       public synchronized void amendCacheConfiguration(String cacheName, ConfigurationBuilder configurationBuilder) {
          super.amendCacheConfiguration(cacheName, configurationBuilder);
          log.tracef("Amend cache configuration for: %s", cacheName);
          if ("entity".equals(cacheName)) {
-            if (ENTITY_CONFIG_COUNTER == NUM_NODES - 1) {
```

Comment:
I'm not sure about this change, based on the class name (`AbstractInjectPartialFailure`) I think it might be important for the test that the operation only fails on one node. Unless "partial" refers to something else @galderz ?